src/common/options.h

```diff
@@ -63,4 +76,26 @@
 void _set_seed_default(tinymt64_t *random_state, const bool choose_seed,
                        const uint64_t seed);
 
+/* Simple set_seed function for the basic generators */
+void _set_seed(struct drand48_data *random_state, const bool choose_seed,
+               const unsigned long long int seed);
+
+/* Output a floating point number r (0.0 < r < 1.0) */
+double generate_random_double(struct drand48_data *random_state);
+
+/* Initialize a data structure used to hold the information required */
+/* by the RNG */
+mca_data_t *get_mca_data_struct(bool *choose_seed, unsigned long long int *seed,
+                                bool *random_state_valid,
+                                struct drand48_data *random_state,
+                                pthread_mutex_t *global_tid_lock,
+                                unsigned long long int *global_tid);
+
+double _mca_rand(mca_data_t *mca_data);
```

Comment:
Why _mca prefix ? This can be used in other backends too.

Reply:
Done.